tasks/fork/maintain-rate.ts

```diff
@@ -85,31 +84,29 @@ HyperdriveDeployBaseTask(
             // value is not recalculated.
 
             // For EETH, we can call rebase on the Etherfi contracts using an
-            // impersonated account.
-            const config = hre.network.config as HttpNetworkUserConfig;
-            config.accounts = "remote";
-            await tc.impersonateAccount({
-                address: MAINNET_ETHERFI_MEMBERSHIP_MANAGER,
-            });
```

Comment:
Why did we get rid of this?

Reply:
not needed and I believe it was causing the multiple config issue.